src/components v2/Atoms/Components/InfoCard/InfoCard.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+  <div
+    ref="container"
+    class="ph-info-card"
+    :class="[
+      !brandBar && 'ph-info-card-non-branded',
+      shadow && 'ph-info-card-shadow',
+      'ph-info-card-border-' + border,
+    ]"
+  >
+    <div v-if="brandBar" ref="brandBar" class="ph-info-card-brand-bar" />
+    <div class="ph-info-card-inner-container">
+      <div v-if="heading" ref="heading" class="ph-info-card-heading">
+        <div class="ph-info-card-gradient-text-container">
+          <PBrandGradientText>
+            <span ref="headingText" class="ph-heading-text">
+              {{ heading }}
+            </span>
+          </PBrandGradientText>
+        </div>
+        <hr class="ph-info-card-hr" />
+
+        <div
+          v-if="icon"
+          class="ph-info-card-icon"
+          :class="'ph-icon-bg-' + iconBg"
+        >
+          <p-icon ref="iconEl" :name="icon" type="xl" />
+        </div>
+      </div>
+
+      <div
+        ref="content"
+        class="ph-info-card-slot"
+        :class="'ph-info-card-slot-bg-' + contentBg"
+      >
+        <slot />
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import PBrandGradientText from '@/components/BrandGradientText';
+import PIcon from '@/components/Icon';
+import { ColorValues } from './types';
+
+export default Vue.extend({
+  name: 'InfoCard',
+
+  components: {
+    PBrandGradientText,
+    PIcon,
+  },
+
+  props: {
+    heading: {
+      type: String as PropType<string>,
+      default: '',
+    },
+
+    icon: {
+      type: String as PropType<string>,
+      default: '',
+    },
+
+    contentBg: {
+      type: String as PropType<ColorValues>,
+      default: 'primary',
+    },
+
+    iconBg: {
+      type: String as PropType<ColorValues>,
+      default: '',
+    },
+
+    shadow: {
+      type: Boolean as PropType<boolean>,
+      default: true,
+    },
+
+    brandBar: {
+      type: Boolean as PropType<boolean>,
+      default: false,
+    },
+
+    border: {
+      type: String as PropType<string>,
+      default: 'transparent',
+    },
+  },
```

Comment:
Please use the export props pattern here.

Reply:
yes, I can't blame anyone but not myself